components/navbar/dropdown-menu.tsx

```diff
@@ -1,6 +1,7 @@
 "use client";
```

Comment:
Do we need this?

Reply:
Yes! It's crucial for letting HeadlessUI work properly, as it internally uses context.